packages/auth/migrations/sqls/20171217200855-profile-up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE users
+  ADD COLUMN "username"     citext unique,
+  ADD COLUMN "hasPublicProfile"  boolean not null default false
+;
```

Comment:
Missing: `CREATE EXTENSION IF NOT EXISTS "citext"`

Reply:
✅